tests/integration/api/test_get_edition.py

```diff
@@ -1,17 +1,19 @@
 import pytest
 import requests
+import json
 from .constants import API_URL
```

Comment:
Ok so the API_URL will have to change depending in the env. I think we can get this from the config file - DRB_API_HOST